src/ui/sass/modules/_SearchBar.scss

```diff
@@ -87,6 +87,7 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
 
     background-color: transparent;
     flex-grow: 1;
```

Comment:
do we still need flex-grow: 1 if we have width: 100% now?

Reply:
no, it's actually not needed

Reply:
updated to just have the width: 100%